pkg/model/serializer.go

```diff
@@ -38,6 +38,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+
+	//ErrDevModeNotValid is raised when development mode in manifest is not 'sync' nor 'hybrid'
+	ErrDevModeNotValid = errors.New("development mode not valid. Value must be one of: ['sync', 'hybrid']")
```

Comment:
do we still want to expose this error outside the pkg?

Reply:
362a5e9417a534ad172bd68ca404ef550df74fa3